Gemfile

```diff
@@ -41,44 +40,45 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 # See miq_expression_spec Date/Time Support examples.
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
```

Comment:
Note, I'm pretty sure this would affect automate ruby method authors if they're concerned with date strings from remote systems.  cc @gmcculloug 

Sorry :koala:, we'll have to go with :us: until using american style dates becomes a problem.

I don't know if there would be any issues or a need to allow this to be configurable.